start-e2e-with-local-env.sh

```diff
@@ -10,5 +10,6 @@
 export E2E_TEST_USER_1_USERNAME=testuser1
 export E2E_TEST_USER_1_PASSWORD=testuser1
 export ZAC_URL=http://host.docker.internal:8080
+export INCLUDE_TAGS=true
```

Comment:
what is this 'INCLUDE_TAGS' param?

Reply:
INCLUDE_TAGS will run cucumber test suite with the provided tags inside ./start-e2e.sh which will exclude all live-env only scenarios (tests running in open forms for example). Perhaps EXCLUDE_LIVE_SCENARIO_TAGS would be a better name

Reply:
nice!